packages/ui/src/Categories.tsx

```diff
@@ -1,29 +1,37 @@
 "use client";
-import { Button } from "./shad/ui/button";
 import { useRecoilState } from "recoil";
+import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from "./shad/ui/select";
 import { category } from "@repo/store";
 
 export const Categories = ({ categories }: { categories: { category: string }[] }) => {
-  function handleFilterButton(category: string) {
+  const [selectedCategory, setSelectedCategory] = useRecoilState(category);
+
+  function handleCategoryChange(category: string) {
     if (category == selectedCategory) {
       setSelectedCategory("");
     } else {
       setSelectedCategory(category);
     }
   }
-  const [selectedCategory, setSelectedCategory] = useRecoilState(category);
+
   return (
-    <div className="flex justify-evenly mx-auto border-2 rounded-full py-1 w-2/3">
-      {categories.map((category) => (
-        <Button
-          key={category.category}
-          variant="ghost"
-          onClick={() => handleFilterButton(category.category)}
-          className={selectedCategory == category.category ? "bg-gray-100 dark:bg-slate-700" : ""}
-        >
-          {category.category}
-        </Button>
-      ))}
+    <div className="flex justify-evenly mx-auto">
+      <Select
```

Comment:
Can we do this conditionally?
If screen width is less than lg, then maybe render the select?

Reply:
yes, on it

Reply:
@hkirat making it less than lg will still have overflow, ill do it at xl?

Reply:
yup